configs/platforms/osx-10.10-x86_64.rb

```diff
@@ -3,8 +3,9 @@
   plat.servicedir '/Library/LaunchDaemons'
   plat.codename "yosemite"
 
-  plat.provision_with 'softwareupdate -i "Command Line Tools (OS X 10.10) for Xcode-7.2"'
-  plat.provision_with 'mkdir /usr/local; cd /usr/local; git clone https://github.com/Homebrew/homebrew.git .; git checkout ab475 -- Library/Formula/boost.rb; /usr/local/bin/brew install pkgconfig'
-  plat.install_build_dependencies_with "PATH=$PATH:/usr/local/bin brew install "
+  plat.provision_with 'mkdir /usr/local; curl http://pl-build-tools.delivery.puppetlabs.net/osx/homebrew.tar.gz | tar -x --strip 1 -C /usr/local -f -; /usr/local/bin/brew tap puppetlabs/brew-build-tools http://pl-build-tools.delivery.puppetlabs.net/osx/brew-build-tools.git --full; /usr/local/bin/brew tap-pin puppetlabs/brew-build-tools'
+  plat.provision_with 'curl -o /usr/local/bin/osx-deps http://pl-build-tools.delivery.puppetlabs.net/osx/osx-deps; chmod 755 /usr/local/bin/osx-deps'
```

Comment:
I grabbed this script and took a look at what it's doing (which is sane), but where does it come from? is it versioned? does it have a deployment/release cycle of its own?